src/components/panel/panel.js

```diff
@@ -2550,3 +2702,28 @@ function getElement(el) {
       document.querySelector(el) : el;
   return angular.element(queryResult);
 }
+
+
+/**
+ * Moves an array element from one array position to another position.
+ * http://stackoverflow.com/questions/5306680
+ *     /move-an-array-element-from-one-array-position-to-another
+ * @param {number} oldIndex The original position of the array element.
+ * @param {number} newIndex The new position that the array element will move
+ *     to.
+ */
+Array.prototype.move = function(oldIndex, newIndex) {
+  while (oldIndex < 0) {
```

Comment:
I don't see a case where we'd use it though. The only place where we use `move` is [here ](https://github.com/angular/material/pull/9538/files#diff-b71bb3e10759daf665e48e9bc558dc99R912) and I don't